### PR TITLE
Add newmembers

### PR DESCRIPTION
**Pull request by fiestared**
@sot528 I added new members' profile. Please check it.

### CODE REVIEW

File: src/yaml/en.yml [hunk 1]
```diff
@@ -143,21 +143,57 @@ member:
...
+      twitter:
+      fb: https://www.facebook.com/yuki.matsuyama.395
+      linkedin:
+    yamashita:
+      id: yamashita
+      name: TAISUKE YAMASHITA
+      body: |
+        After graduating from Tokyo Denki University Faculty of Engineering in 2008, Yamashita joined a banking service development company where he was involved in customizing packages for regional banks, transitioning to a joint center operations. Interested in the development of open systems rather than the development of general purpose systems, in 2012, Yamashita changed to a web development company. He was involved in the development and operation of the LAMP environment. In 2017 he began work as a freelance contractor. At ALIS he mainly handles the development of authentication and search functions and tools.
+      role: ENGINEER
+      twitter: https://github.com/yaasita
```
Comment (sot528) [inline src/yaml/en.yml]: It looks GitHub's URL.
NOT Twitter's URL.

File: src/yaml/ja.yml [hunk 2]
```diff
@@ -150,21 +150,57 @@ member:
...
+      twitter:
+      fb: https://www.facebook.com/yuki.matsuyama.395
+      linkedin:
+    yamashita:
+      id: yamashita
+      name: TAISUKE YAMASHITA
+      body: |
+        東京電機大学工学部を2008年卒業後、銀行系サービス開発会社に入社。地方銀行向けパッケージのカスタマイズ、共同センター移行、運用に携わる。汎用系の開発よりオープン系の開発に興味を持ったため、2012年より、Web系開発会社に転職。LAMP環境の開発・運用に携わる。2017年よりフリーランス。ALISでは主に認証部分、検索部分の開発を担当。
+      role: ENGINEER
+      twitter: https://github.com/yaasita
```
Comment (sot528) [inline src/yaml/ja.yml]: It looks GitHub's URL.
NOT Twitter's URL.

File: src/team.pug [hunk 3]
```diff
@@ -159,79 +175,168 @@ body
...
-    member.profile.kamei.twitter, 
-    member.profile.kamei.fb, 
+    member.profile.kamei.role,
+    member.profile.kamei.body,
+    member.profile.kamei.twitter,
+    member.profile.kamei.fb,
     member.profile.kamei.linkedin)
   .p-popup-overlay(data-exit=member.profile.kamei.id)
 
   article.p-popup.p-popup-member.is-effect(src='/img/pc/img_profile_usagi.png', id="js-popup-"+member.profile.usagi.id)
```
Comment (sot528) [inline src/team.pug]: ~~img_profile_usagi~~ => **img_profile_akagi**

**Comment by sot528** [comment]
Mrs. Kubori's first name is "CHISA". 
NOT "CHIKA"

Please fis it 🙏

**Comment by sot528** [comment]
I can not see member faces when select Japanese language.
Maybe it depends image file path.

**Comment by fiestared** [comment]
@sot528 I fixed all points you mentioned, but somehow I couldn't fix the Akagi's profile picture( I changed the profile path to /img/pc/img_profile_akagi.png , but still shown as a img_profile_usagi.png

**Comment by sot528** [comment]
@fiestared 
> I couldn't fix the Akagi's profile picture
Here you are ;)
- https://github.com/AlisProject/welcome/blob/master/src/mixin/popup_member.pug#L6

Actually, it it no good implementation...

**Comment by sot528** [comment]
I fixed it.
LGTM!

**Pull request merged**